core.js

```diff
@@ -66,7 +66,7 @@ module.exports = function (xhr) {
       }
 
       // Ensure that we have the appropriate request data.
-      if (options.data == null && model && (method === 'create' || method === 'update' || method === 'patch')) {
+      if (options.data == null && model && (type === 'POST' || type === 'PUT' || type === 'PATCH')) {
```

Comment:
since we already have `lodash.includes`, would this line be more readable as:

```js
if (options.data == null && model && includes(['POST', 'PUT', 'PATCH'], type)) {
```

Similarly on [L91](https://github.com/ruiramos/ampersand-sync/blob/master/core.js#L91)